.github/workflows/test.yml

```diff
@@ -77,3 +77,26 @@ jobs:
       - run: CI=false yarn build
         env:
           CI: true
+
+  run-dotrun:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
```

Comment:
This is full of empty lines unlike the rest of the file. Probably best to keep it consistent.